package.json

```diff
@@ -6,9 +6,7 @@
   "scripts": {
     "compile": "npx hardhat compile",
     "test": "npx hardhat test",
-    "coverage": "npx hardhat coverage --network hardhat",
-    "deploy": "npx hardhat run scripts/deploy.ts",
-    "deploy:staging": "npx hardhat run scripts/deploy_staging.ts"
```

Comment:
Whats the reasoning for these to be removed in favour of `npx hardhat run ...`?

Reply:
Network parameter has to be passed like this: `npm run deploy -- --network kovan`